protobuf-format.md

```diff
@@ -0,0 +1,88 @@
+# Protobuf Event Format for CloudEvents  - Version 0.1
+
+## Abstract
+
+The Protocol Buffers Format for CloudEvents (CE) defines
+the encoding of events in the protocol buffers binary format.
+
+## Status of this document
+
+This document is a working draft.
+
+## 1. Introduction
+
+CloudEvents is a standardized and transport-neutral definition of the
+structure and metadata description of events. This specification
+defines how the [Context
+Attributes](spec.md#context-attributes) defined in the CloudEvents in
+the protocol buffers binary encoding MUST BE encoded. Transcoding to
+and from other formats (e.g. JSON) is out of the scope of this
+proposal.
```

Comment:
s/proposal/document/   (or specification)
since once its merged it won't be a proposal any more